lambda_functions/alerter/alerter.py

```diff
@@ -20,6 +16,8 @@ def lambda_handler(event, context):
     :type context: N/A
     :return: Return a success or failure outcome on whether the slack message was sent
     :rtype: json
+    :raises Exeption: will raise a runtime error if appropriate
```

Comment:
Replace "if appropriate" with the scenario when a the exception is raised.

Reply:
done :)